packages/mongo-nav/src/user-menu/UserMenu.tsx

```diff
@@ -459,14 +481,41 @@ function UserMenu({
             {isGovernment ? 'Cloud for Government' : 'Cloud'}
           </MenuItem>
         )}
+
+        <MenuItem
+          {...subMenuContainer.prop}
+          {...sharedProps}
+          size="large"
+          active={isDocs}
+          disabled={!account}
+          href="https://docs.mongodb.com"
+          glyph={<DocsIcon {...ariaHiddenProps} />}
+          className={cx(
+            subMenuContainerStyle,
+            {
+              [subMenuActiveContainerStyle]: isDocs,
+            },
+            css`
+              background-color: ${uiColors.gray.light3};
+              border-top: 1px solid ${uiColors.gray.light2};
+            `,
```

Comment:
we also use this style below. let's pull it out and give it a name

Reply:
👍